### ISSUE TITLE
requests for missing files never return

### ISSUE THREAD

**Issue by also** [tailrecursion/lein-simpleton#3]
Running

```
lein simpleton 8000
```

and (assuming there's no file named nope)

```
curl http://localhost:8000/nope
```

never returns. Simpleton logs

```
./nope (No such file or directory)
```

**Comment by mynomoto** [comment]
I think that this is the culprit:
https://github.com/tailrecursion/lein-simpleton/blob/master/src/leiningen/simpleton.clj#L107-L110
It probably should return a 404.

**Comment by fogus** [comment]
@mynomoto Thanks for the pointer.  The fix has been applied to the 1.3.0 version at https://clojars.org/lein-simpleton.
